### ISSUE TITLE
Talent not selected on Load Options

### ISSUE THREAD

**Issue by hasore** [WeakAuras/WeakAuras2#245]
Should be good if WeakAuras implement a Talent not Selected option on Load Options

Edited Prototypes.lua for local support it
After function WeakAuras.CheckTalentByIndex
```lua
function WeakAuras.CheckTalentUnselectedByIndex(index)
  return WeakAuras.CheckTalentByIndex(index) == false;
end```

Before talent and talent2 args
```lua
    {
      name = "talentunselected",
      display = L["Talent not selected"],
      type = "multiselect",
      values = valuesForTalentFunction,
      test = "WeakAuras.CheckTalentUnselectedByIndex(%d)",
    },
```

**Comment by InfusOnWoW** [comment]
I don't think this adds much. As you can select the other two talents on the same row and get the same effect. Any particular reason for adding it?

**Comment by hasore** [comment]
New legendary rings and shared talents between specializations, an example for Priest:
1 - Equip Soul of High Priest on discipline specialization and choose a secondary talent besides Twist of Fate.
2 - Switch to shadow specialization, remove ring (auto via Equipament Manager for example) and you will not have any of the three talents on Twist of Fate row.

Same happen for other classes with shared talents between specialization.
